Cargo.toml

```diff
@@ -81,7 +81,8 @@ unnecessary_struct_initialization = "allow"
 use_self = "allow"
 
 [workspace.dependencies]
-alloy = { git = "https://github.com/alloy-rs/alloy", rev = "27d5ba9", features = [
+alloy = { git = "https://github.com/zerosnacks/alloy", branch = "fix/alloy-provider-prelude", features = [
```

Comment:
@zerosnacks reminder to replace this post https://github.com/alloy-rs/alloy/pull/436 merge and pin to commit

Reply:
No longer blocked